component/api/src/main/java/org/exoplatform/social/notification/service/SpaceWebNotificationService.java

```diff
@@ -85,5 +85,14 @@ public interface SpaceWebNotificationService {
    * @return           Map of application and unread items
    */
   Map<String, Long> countUnreadItemsByApplication(long creatorId, long spaceId);
+  
+  /**
+   * Get a list of unread items per space to a given {@link MetadataItem}
+   *
+   * @return Map of space id and unread items
+   */
+  Map<Long, Long> countUnreadItemsBySpace(String authenticatedUser);
```

Comment:
Missing Javadoc for `username` parameter

```suggestion
  Map<Long, Long> countUnreadItemsBySpace(String username);
```

Reply:
Done!